qcommon/test/org/qcmg/common/util/PileupElementLiteUtilTest.java

```diff
@@ -186,45 +182,45 @@ public void testPassesCountCheckSecondPass() {
 		
 	}
 	
-	@Test
-	public void toSummaryString() {
-		Accumulator accum = new Accumulator(1);
-		accum.addBase((byte)'A', (byte) 1, true, 1, 1, 2, 1);
-		assertEquals("A1[1],0[0]", PileupElementLiteUtil.toSummaryString(accum.getLargestVariant('\u0000'), "A"));
-		accum.addBase((byte)'T', (byte) 1, true, 1, 1, 2, 1);
-		assertEquals("T1[1],0[0]", PileupElementLiteUtil.toSummaryString(accum.getLargestVariant('\u0000'), "T"));
-	}
-	@Test
-	public void toOABS() {
-		Accumulator accum = new Accumulator(1);
-		accum.addBase((byte)'A', (byte) 1, true, 1, 1, 2, 1);
-		assertEquals("A1[1]0[0]", PileupElementLiteUtil.toObservedAlleleByStrand(accum.getLargestVariant('\u0000'), "A"));
-	}
-	
-	@Test
-	public void testGetLargestVariantNovelStarts() {
-		Accumulator accum = new Accumulator(1);
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'A'));
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'C'));
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'G'));
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'T'));
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'N'));
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'X'));
-		
-		accum.addFailedFilterBase((byte)'A');
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'A'));
-		byte A = 'A';
-		accum.addBase(A, (byte) 1, true, 1, 1, 2, 1);
-		assertEquals(0, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'A'));
-		assertEquals(1, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'C'));
-		byte C = 'C';
-		accum.addBase(C, (byte) 1, true, 1, 1, 2, 1);
-		accum.addBase(C, (byte) 1, true, 2, 1, 2, 1);
-		assertEquals(1, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'C'));	// still have the A in there
-		assertEquals(2, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'A'));
-		assertEquals(2, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'G'));
-		assertEquals(2, PileupElementLiteUtil.getLargestVariantNovelStarts(accum, 'T'));
-	}
+//	@Test
```

Comment:
I see there are several commented-out tests in this ci; is that because they might be resurrected in some form?